lib/vmdb/loggers.rb

```diff
@@ -50,6 +54,16 @@ def self.create_logger(log_file_name, logger_class = VMDBLogger)
       configure_external_loggers
     end
 
+    private_class_method def self.create_null_logger(log_file_name, logger_class)
+      logger_class.new(nil).tap do |logger|
```

Comment:
instead of `nil` can you pass in a parent of `ActiveSupport::Logger.broadcast(log_broadcast_target)`?

think that is the same thing as `logger.extend(parent)`

Reply:
I'm tempted to drop the null logger entirely and just have multiple instances of the container logger or the journald logger.